src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpCwdsDAO.scala

```diff
@@ -57,6 +60,35 @@ class HttpCwdsDAO(enabled: Boolean) extends CwdsDAO {
       .toList
   }
 
+  override def importV1(workspaceId: String,
+                        asyncImportRequest: AsyncImportRequest
+                       )(implicit userInfo: UserInfo): GenericJob = {
+    val importApi: ImportApi = new ImportApi()
+    importApi.setApiClient(getApiClient(userInfo.accessToken.token))
+
+    importApi.importV1(toCwdsImportRequest(asyncImportRequest), UUID.fromString(workspaceId))
+  }
+
+  protected[dataaccess] def toCwdsImportRequest(asyncImportRequest: AsyncImportRequest): ImportRequest = {
+    val importRequest: ImportRequest = new ImportRequest
+    importRequest.setUrl(URI.create(asyncImportRequest.url))
+    importRequest.setType(toCwdsImportType(asyncImportRequest.filetype))
+
+    // as of this writing, the only available option is "tdrSyncPermissions"
+    asyncImportRequest.options.map { opts =>
+      opts.tdrSyncPermissions.map { tdrSyncPermissions =>
+        importRequest.setOptions(Map[String, Object]("tdrSyncPermissions" -> tdrSyncPermissions.asInstanceOf[Object]).asJava)
+      }
+    }
+
+    importRequest
+  }
+
+  protected[dataaccess] def toCwdsImportType(input: String): ImportRequest.TypeEnum = {
+    TYPE_TRANSLATION.getOrElse(input,
+      throw new FireCloudException("Import type unknown; possible values are: " + TYPE_TRANSLATION.keys.mkString))
+  }
```

Comment:
this throws an exception if it can't translate, as compared to `toImportServiceStatus` which returns a default value. The difference here is that cWDS only supports a known set of `TypeEnum` values and if we send a default, it'll cause an error downstream; the status value from `toImportServiceStatus` is mostly informational.